### PR TITLE
[FE] apply user session to updateDBPassword api

### PR DESCRIPTION
**Pull request by kimong1**
Actually, session middleware had been already set up. (Of course because we have login and logout api)
image

Because of the middleware above, on the server side, session data can be read/written simply like this:
image

So I just used this to `updateDBPassword` api.
Review please

### CODE REVIEW

File: src/routes/userRoutes.js [hunk 1]
```diff
@@ -92,14 +92,14 @@ routes.loginUser = async (req, res) => {
   }
 }
   routes.updateDBPassword = async(req,res) =>{
-    if(!req.params.id ||!req.body.password ){
+    if(!req.session.username ||!req.body.password ){
```
Comment (greenc123) [inline src/routes/userRoutes.js:95]: This is being a little nitpicky but try doing:

```
const username = req.session.username
const password = req.body.password
```

Comment (kimong1) [reply src/routes/userRoutes.js]: Done!

File: src/routes/userRoutes.js [hunk 2]
```diff
@@ -1,143 +1,178 @@
...
-      const updatedAccount = await setDBPassword(userAccount,req.body.password) 
-      logger.info(`user ${userAccount.id} updates password`)
-      return res.status(200).json({...updatedAccount.dataValues,password:null})
-    } catch (err) {
-      logger.error('Password update failed. Please try again',req.params.id, err)
-      return res.status(500).json({error: {message: 'Password update failed. Please try again'}})
-    }
+};
+routes.updateDBPassword = async (req, res) => {
+  const username = req.session.username;
```
Comment (kimong1) [inline src/routes/userRoutes.js:121]: This is the part that is actually changed. Other changes are just formatting

**Comment by kimong1** [comment]
This PR also include auto-formatting. I made a coment where is actually changed

**Pull request merged**